internal/spentcalories/spentCalories.go

```diff
@@ -12,7 +16,24 @@ const (
 )
 
 func parseTraining(data string) (int, string, time.Duration, error) {
-	// ваш код ниже
+	parts := strings.Split(data, " ")
+	if len(parts) != 3 {
+		return 0, "", 0, errors.New("некорректный формат данных")
+	}
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, "", 0, err
```

Comment:
Можно оборачивать ошибку, чтобы был красивый вывод
```suggestion
		return 0, "", 0, fmt.Errorf("conversion error: %w", err)
```